src/components/tile-select/tile-select.stories.mdx

```diff
@@ -0,0 +1,217 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { useState } from 'react';
+
+import { TileSelect, TileSelectGroup } from '.';
+import Pill from "../pill/"
+import Icon from '../icon';
+
+<Meta
+  title="Design System/Tile Select"
+  parameters={{ info: { disable: true }}}
+/>
+
+# TileSelect
+Is a deselectable radio button input visualized as a single or grouped set of tiles
```

Comment:
Just a small adjustment to this line. I think it would be best to specify what is used in the example rather than just "Is". Also full stop required at the end of the line. Something along the lines of:

"Tile Select is a deselectable radio button input visualized as a single or grouped set of tiles."